pkg/agent/packetcapture/capture/bpf.go

```diff
@@ -56,6 +59,18 @@ var ProtocolMap = map[string]uint32{
 	"ICMP": 1,
 }
 
+var ICMPMsgTypeMap = map[crdv1alpha1.ICMPMsgType]uint32{
+	crdv1alpha1.ICMPMsgTypeEcho:       8,
+	crdv1alpha1.ICMPMsgTypeEchoReply:  0,
+	crdv1alpha1.ICMPMsgTypeDstUnreach: 3,
+	crdv1alpha1.ICMPMsgTypeTimexceed:  11,
+}
+
+type message struct {
```

Comment:
`message` is much too generic for a type name in this case
it seems that this type is specific to the ICMP case?

Reply:
not addressed yet